scripts/workspace_deploy.py

```diff
@@ -126,13 +133,14 @@
     # create file with automatic redirect
     if hasattr(config, 'common_outputs_client') and getattr(config, 'common_outputs_client'):
         clientFilePath = os.path.join(getattr(config, 'common_outputs_directory'), getattr(config, 'common_outputs_client'))
-        try:
-            with open(clientFilePath, "w") as clientFile:
-                clientFile.write('<meta http-equiv="refresh" content=\"0; url=' + clientv2URL + '\" />')
-                clientFile.write('<p><a href=\"' + clientv2URL + '\">Redirect</a></p>')
-            clientFile.close()
-        except IOError:
-            eprintf('ERROR: Cannot write to %s\n', clientFilePath)
-            sys.exit(1)
+        if hasattr(config, 'context_client_name'):
```

Comment:
there can be unassigned variables without hasattr(config, 'context_client_name') condition

Reply:
Merged.